src/command_modules/azure-cli-advisor/azure/cli/command_modules/advisor/tests/test_advisor_commands.py

```diff
@@ -0,0 +1,63 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+
+from azure.cli.testsdk import LiveScenarioTest, ScenarioTest, ResourceGroupPreparer
+
+
+class AzureAdvisorLiveScenarioTest(LiveScenarioTest):
+
+    def test_list_disable_enable_recommendations(self):
+        output = self.cmd('advisor recommendation list').get_output_in_json()
+        self.assertTrue(len(output) > 1)
+        output = self.cmd('advisor recommendation list --category cost').get_output_in_json()
+        self.assertTrue(len(output) > 0)
```

Comment:
Recommend https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertGreater as the error will be clearer that way.
Same elsewhere.